### PR TITLE
ViewPager and SectionsPagerAdapter

### PR DESCRIPTION
**Pull request by balintvecsey**
Branched from balint-settingsfragment.

### CODE REVIEW

File: app/src/main/java/com/greenfox/opal/gitinder/MainActivity.java [hunk 1]
```diff
@@ -1,105 +1,104 @@
...
-        });
+    onLogin("Bond", "abcd1234");
+    onLogin("", "");
+
+    checkLogin();
+  }
+
+  public void setupViewPager(ViewPager viewPager) {
+    SectionsPagerAdapter adapter = new SectionsPagerAdapter(getSupportFragmentManager());
+    adapter.addFragment(new SwipingFragment(), "Swiping");
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/opal/gitinder/MainActivity.java]: @balintvecsey strings should resources or constants

File: app/src/main/java/com/greenfox/opal/gitinder/service/SectionsPagerAdapter.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SectionsPagerAdapter extends FragmentPagerAdapter {
+
+  private static final List<Fragment> mFragmentList = new ArrayList<>();
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/opal/gitinder/service/SectionsPagerAdapter.java]: @balintvecsey this shouldnt be static final

Comment (adamgyulavari) [reply app/src/main/java/com/greenfox/opal/gitinder/service/SectionsPagerAdapter.java]: initialization should be in constructor

File: app/src/main/res/layout/fragment_matches.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  android:orientation="vertical" android:layout_width="match_parent"
+  android:layout_height="match_parent">
+
+</LinearLayout>
```
Comment (adamgyulavari) [inline app/src/main/res/layout/fragment_matches.xml]: @balintvecsey new line at the end of file

File: app/src/main/res/menu/menu_main_activity_tabbed.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:context="com.greenfox.opal.gitinder.MainActivity" >
+    <item android:id="@+id/action_settings"
+        android:title="@string/action_settings"
+        android:orderInCategory="100"
+        app:showAsAction="never" />
+</menu>
```
Comment (adamgyulavari) [inline app/src/main/res/menu/menu_main_activity_tabbed.xml]: @balintvecsey this file is unnecessary

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

**Review by adamgyulavari — APPROVED** [review]

**Comment by adamgyulavari** [comment]
@balintvecsey resolve conflicts

**Comment by balintvecsey** [comment]
Not yet complete, because after I solved conflicts, I started to work on to join the fragments together, and I found some bug, just like on development branch earlier.

**Comment by balintvecsey** [comment]
Everything is ok now :)

**Pull request merged**